build.gradle

```diff
@@ -360,7 +360,7 @@ for ( sv in ['2_10', '2_11', '2_12'] ) {
 }
 
 def connectPkgs = ['connect:api', 'connect:runtime', 'connect:json', 'connect:file']
-def pkgs = ['clients', 'examples', 'log4j-appender', 'tools', 'streams', 'streams:examples'] + connectPkgs
+def pkgs = ['clients', 'examples', 'log4j-appender', 'tools', 'streams', 'streams:examples', 'streams:upgrade-system-tests-0100'] + connectPkgs
```

Comment:
We should exclude this from `installAll`, `releaseTarGzAll` and `uploadArchivesAll` otherwise it would be uploaded to maven. I think we'd better always jar locally before test than trying to pull from the messed repo.